include/LLGL/Container/STLAllocator.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * STL Allocator.h
+ *
+ * Copyright (c) 2015 Lukas Hermanns. All rights reserved.
+ * Licensed under the terms of the BSD 3-Clause license (see LICENSE.txt).
+ */
+
+#ifndef LLGL_STL_ALLOCATOR_H
+#define LLGL_STL_ALLOCATOR_H
+
+#ifndef LLGL_CUSTOM_ALLOCATOR
+#include <memory>
+#endif
+
+namespace LLGL{
+
+#ifndef LLGL_CUSTOM_ALLOCATOR
+
+template < typename Type >
+using llgl_allocator = std::allocator< Type >;
```

Comment:
If we go the route with the `STL` namespace, I would also just put it there and call it with the same naming convention as the other aliases, i.e. `STL::Allocator`.